randomizer.py

```diff
@@ -593,30 +616,30 @@ def potionCards(self):
 Empires = Set("Empires")
 Empires.AddCards(
     [
-        "Engineer",
-        "City Quarter",
-        "Overlord",
-        "Royal Blacksmith",
-        "Encampment/Plunder",
-        "Patrician/Emporium",
-        "Settlers/Bustling Village",
         "Castles",
-        "Catapult/Rocks",
-        "Chariot Race",
-        "Enchantress",
-        "Farmers' Market",
-        "Gladiator/Fortune",
-        "Sacrifice",
-        "Temple",
-        "Villa",
-        "Archive",
         "Capital",
         "Charm",
-        "Crown",
-        "Forum",
-        "Groundskeeper",
-        "Legionary",
-        "Wild Hunt",
+        {"name": "Engineer", "types": {Action}},
+        {"name": "City Quarter", "types": {Action}},
+        {"name": "Overlord", "types": {Action}},
+        {"name": "Royal Blacksmith", "types": {Action}},
+        {"name": "Encampment/Plunder", "types": {Action}},
+        {"name": "Patrician/Emporium", "types": {Action}},
+        {"name": "Settlers/Bustling Village", "types": {Action}},
+        {"name": "Catapult/Rocks", "types": {Action}},
```

Comment:
Is Rocks eligible for Obelisk?

Reply:
Yes, the eligibility is based on the randomizer card, which is an action for all of the 4x split piles.